minecraft_servers/BeeGames.eu/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+   "server_name":"BeeGames.eu",
+   "nice_name":"BeeGames.eu",
+   "direct_ip":"mc.beegames.eu",
+   "server_wildcards":[
+      "%.beegames.eu"
+   ],
+   "supported_languages":[
+      "cs",
+      "sk"
+   ],
+   "social":{
+      "web":"https://beegames.eu/",
+      "web_shop":"https://store.beegames.eu/",
+      "web_support":"https://wiki.beegames.eu/",
+      "tiktok":"BeeGames.eu",
+      "facebook":"BeeGames.eu",
+      "instagram":"beegames.eu",
+      "discord":"https://discord.com/invite/kZG4NQAMJE",
+      "teamspeak":"ts3.beegames.eu"
```

Comment:
Please remove the / at the end of every url here

Reply:
Hello @JakobBruening,

I can´t change a name of direcotry. But the other things I changed. Thanks.